wpeverest/everest-forms
# Fix - EVF Tooltip issue with WooCommerce orders table

### All Submissions:



### Changes proposed in this Pull Request:
This PR fixes the EVF Tooltip issue with the WooCommerce orders table


### How to test the changes in this Pull Request:

1. Install WooCommerce.
2. Create an order with payment completed status and hover.
3. Hover the Payment completed and verify the issue is solved or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - EVF Tooltip issue with WooCommerce orders table.
